{
  "cveTags": [],
  "descriptions": [
    {
      "lang": "en",
      "value": "Rejected reason: DO NOT USE THIS CVE RECORD. ConsultIDs: none. Reason: This record was withdrawn by its CNA. Further investigation showed that it was not a security issue. Notes: none."
    }
  ],
  "id": "CVE-2024-36699",
  "lastModified": "2024-06-14T15:15:50.967",
  "metrics": {},
  "published": "2024-06-12T14:15:12.100",
  "references": [],
  "sourceIdentifier": "cve@mitre.org",
  "vulnStatus": "Rejected"
}